notebooks/book1/02/change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
<p>Add meaning ful title in 1st markdown cell, see other closed PR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Check two times all the comments and if it resolved then click on <code>resolve comment</code>   button</p><p>This comment is left</p><p> </p>